pkg/trace/transform/transform.go

```diff
@@ -146,7 +146,7 @@ func OtelSpanToDDSpan(
 		default:
 			// Exclude Datadog APM conventions.
 			// These are handled below explicitly.
-			if k != "http.method" && k != "http.status_code" {
+			if k != "http.method" && k != "http.status_code" && k != "service.name" && k != "operation.name" && k != "resource.name" && k != "span.type" {
 				SetMetaOTLP(ddspan, k, value)
```

Comment:
You'll want to update SetMetaOTLP as well

Reply:
I decided to leave SetMetaOTLP as is since `convertSpan` (the mapper for ReceiveResourceSpansV1) [needs Service, Name etc. to be set from attributes.](https://github.com/DataDog/datadog-agent/blob/47294c1176849a76403aa1b5f0ce2b95c7c93614/pkg/trace/api/otlp.go#L534) If I'm going to change it, it might make sense to create one separate function for each of v1/v2